src/AccountingService.php

```diff
@@ -938,20 +1140,6 @@ public function numberOfEmployees($numberOfEmployees)
         return $this->setProperty('numberOfEmployees', $numberOfEmployees);
     }
 
-    /**
```

Comment:
This type is an organization only making offers but not getting offered.
https://schema.org/AccountingService